### PR TITLE
issue #1 

### PR DESCRIPTION
**Pull request by andreiSaw**
initialize model with config file yaml

### CODE REVIEW

File: modelgym/model.py [hunk 1]
```diff
@@ -31,3 +44,17 @@ def fit(self, params, dtrain, dtest, n_estimators):
 
     def predict(self, bst, dtest, X_test):
         raise NotImplementedError('Method predict is not implemented.')
+
+    def load_config(self, filepath):
+        if os.path.exists(filepath):
+            import yaml
+            with open(filepath) as f:
```
Comment (anaderi) [inline modelgym/model.py]: move import to the top of the file

File: tests/test_model.py [hunk 2]
```diff
@@ -96,3 +94,13 @@ def test_predict(preprocess_data):
     roc_auc = ans['roc_auc']
     print("ROC_AUC: ", roc_auc)
     assert roc_auc <= MAX_ROC_AUC_SCORE
+
+
+def test_load_and_save():
+    model = modelgym.Model()
+    filepath = "data/newtree.yaml"
```
Comment (anaderi) [inline tests/test_model.py]: no such file.. make fixture?

File: tests/test_model.py [hunk 3]
```diff
@@ -96,3 +94,13 @@ def test_predict(preprocess_data):
...
+    assert os.path.exists(filepath)
+    model.load_config(filepath)
```
Comment (anaderi) [inline tests/test_model.py]: should also check that values from loaded model config (self.space, etc) match values from the fixure.

**Comment by anaderi** [comment]
one more thing: add pyyaml module to https://github.com/yandexdataschool/modelgym/blob/master/environment/environment.yaml

**Comment by andreiSaw** [comment]
corrected

File: tests/test_model.py [hunk 4]
```diff
@@ -97,10 +97,23 @@ def test_predict(preprocess_data):
 
 
 def test_load_and_save():
-    model = modelgym.Model()
+    model = modelgym.Model(learning_task=TASK_CLASSIFICATION)  # model to save and then read
     filepath = "data/newtree.yaml"
```
Comment (anaderi) [inline tests/test_model.py]: would be really bad if user had this file (data/newtree.yaml) before running the test... instead use tempfile.mkstemp(suffix=".test.yaml") or TemporaryFile
https://security.openstack.org/guidelines/dg_using-temporary-files-securely.html

**Comment by andreiSaw** [comment]
added tempfile.NamedTemporaryFile usage

**Pull request merged**